app/src/main/res/layout-land/all_topics_item.xml

```diff
@@ -6,7 +6,7 @@
     android:layout_height="wrap_content">
 
     <TextView
-      android:id="@+id/all_topics_text_view"
+      android:id="@+id/all_topics_item_all_topics_heading_text_view"
```

Comment:
I am not so convinced with this type of naming conventions. @BenHenning @rt4914 @nikitamarysolomanpvt   I would rather suggest `all_topics_item_heading_text_view` would make it much understandable rather than having redundant names. Please let me know what you think.